esx_service/utils/auth_api.py

```diff
@@ -785,9 +852,8 @@ def check_privilege_parameters(privilege):
 
     return None
 
-
 @only_when_configured()
-def _tenant_access_add(name, datastore, allow_create=None, default_datastore=False,
+def _tenant_access_add(name, datastore, allow_create=None,
```

Comment:
Just asking, can datastore be the default DS? Is it possible to add and remove privileges to the default DS?